internal/configs/module_deprecations.go

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package configs
```

Comment:
Don't forget to add tests for `module_deprecations_test.go` 👍

Reply:
Included 👍 https://github.com/hashicorp/terraform/pull/34943/commits/dcd574261489c5e2a3709f5b592a0a6093ee8e03